packages/opentelemetry-sdk-trace-base/src/BasicTracerProvider.ts

```diff
@@ -255,16 +235,4 @@ export class BasicTracerProvider implements TracerProvider {
       });
     }
   }
-
-  protected _buildExporterFromEnv(): SpanExporter | undefined {
```

Comment:
so IIUC it means now `BasicTraceProvider` wont' look up to env to resolve the exporter and it is mandatory to pass it into the constructor right?

This means `NodeSDK` or any other component creating the provider is responsible to resolve the exporter from env if applies right?

Reply:
> so IIUC it means now BasicTraceProvider wont' look up to env to resolve the exporter and it is mandatory to pass it into the constructor right?
>
> This means NodeSDK or any other component creating the provider is responsible to resolve the exporter from env if applies right?


Yep - that's the solution I'd propose. 🙂 

This PR is just a proposal, so we don't have to do it, but I feel like this is a cleaner approach as we'd eliminate an additional (confusing) way to do accomplish the same thing. 

Neither the `LoggerProvider` nor `MeterProvider` have equivalent functionality and I'm not aware of any requests that I remember that asked for adding it. 🤔  Removing it also has the bonus of aligning functionality between the three signals. I'd like to align the three signals as much as possible to give a more consistent experience to end-users.